src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -343,8 +347,12 @@ private static AnalyzeJobOutput analyzeRepos(String outputPath, List<RepoConfigu
         updateAuthorList(config);
         updateIgnoreCommitList(config);
 
-        AuthorshipSummary authorshipSummary = AuthorshipReporter.generateAuthorshipSummary(config);
+        if (config.isFindingPreviousAuthorsPerformed()) {
+            generateIgnoreRevsFile(config);
+        }
+
         CommitContributionSummary commitSummary = CommitsReporter.generateCommitSummary(config);
+        AuthorshipSummary authorshipSummary = AuthorshipReporter.generateAuthorshipSummary(config);
```

Comment:
I have found the bug, it appears that I have accidentally switched the positions of generating the commitContributionSummary and the authorship summary when merging PRs in.